Add method for affiliation analysis (#35)
greenelab/iscb-diversity-manuscript: content/20.results.md

  For each region, the mean predicted probability of Pubmed articles is shown as teal LOESS curve, and the mean probability and 95% confidence interval of the ISCB honoree predictions are shown as dark circles and vertical lines (B).

](https://raw.githubusercontent.com/greenelab/iscb-diversity/master/figs/region_breakdown.png){#fig:region_breakdown}

> ### Assessing Affiliation Diversity of Authors and Honorees

Reviewer: ```suggestion
### Assessing Affiliation Diversity of Authors and Honorees

```
I think this needs a newline, unless headers are special.

> We observed an increasing proportion of author affiliation located in East Asia & Pacific (Fig. {@fig:affiliation_breakdown}A, left).

Reviewer: ```suggestion
We observed an increasing proportion of author affiliations in East Asia & the Pacific (Fig. {@fig:affiliation_breakdown}A, left).
```

> This was primarily driven by publications in Bioinformatics and BMC Bioinformatics (see [notebook](https://greenelab.github.io/iscb-diversity/15.analyze-affiliation.html#sup_fig_s6)).

Reviewer: ```suggestion
This was primarily driven by publications in _Bioinformatics_ and _BMC Bioinformatics_ (see [notebook](https://greenelab.github.io/iscb-diversity/15.analyze-affiliation.html#sup_fig_s6)).
```
I think we've been going italics

Author: ah yes sorry it's been a while

> Among ISCB keynote speakers and fellows, we found that the majority of honorees are affiliated with institutions and companies from North America (Fig. {@fig:affiliation_breakdown}A, right).

Reviewer: ```suggestion
Among ISCB keynote speakers and fellows, we found that the majority of honorees were affiliated with institutions and companies from North America (Fig. {@fig:affiliation_breakdown}A, right).
```

> Noting the small sample size, we found discrepancies between the group of honoree composition versus PubMed. 

Reviewer: ```suggestion
We observed discrepancies between the affiliations of honorees and article authors.
```

> Compared to Pubmed authors, ISCB keynote speakers and fellows have a higher proportion of individuals affiliated with North American institutions and companies (Fig. {@fig:affiliation_breakdown}B) and lower proportion of individuals affiliated with institutions or companies from Europe & Central Asia and East Asia & Pacific. 

Reviewer: ```suggestion
ISCB keynote speakers and fellows had a higher proportion of individuals affiliated with North American institutions and companies (Fig. {@fig:affiliation_breakdown}B) and lower proportion of individuals affiliated with institutions or companies from Europe & Central Asia and East Asia & Pacific than authors. 
```

Author: I left "Compared to authors" at the beginning of this sentence for more direct reading. I fixed the tense as you suggested.

Reviewer: 👍

> ![Compared to Pubmed authors, honorees affiliated with institutions from North America are overrepresented while those from Europe and Central Asia, East Asia & Pacific group are underrepresented.

Reviewer: ```suggestion
![Compared to authors, honorees affiliated with institutions from North America were overrepresented while those from Europe and Central Asia, East Asia & Pacific group were underrepresented.
```

> Estimated composition of affiliation locations over the years of (A, left) all Pubmed computational biology and bioinformatics journal authors, and (A, right) all ISCB Fellows and keynote speakers was computed as the geographical proportions of Pubmed articles or ISCB honorees affiliations each year.

Reviewer: ```suggestion
We estimated the composition of affiliation locations each year for (A, left) computational biology and bioinformatics journal authors, and (A, right) all ISCB Fellows and keynote speakers.
```

> For each region, the geographical proportions of Pubmed articles affiliations are shown as teal LOESS curves, and the geographical proportions and 95% confidence interval of the ISCB affiliations are shown as dark circles and vertical lines (B).

Reviewer: ```suggestion
(B) For each region, we plotted the geographical proportions of article authors as teal LOESS curves, and ISCB honorees are shown as dark circles and vertical lines (error indicators show 95% confidence intervals).
```

> ](https://raw.githubusercontent.com/greenelab/iscb-diversity/master/figs/region_breakdown.png){#fig:affiliation_breakdown}

Reviewer 2: What do the letters correspond to in this figure? The country groupings in

- [`2020-01-31_natural-earth-world-map.png`](https://raw.githubusercontent.com/greenelab/iscb-diversity/b41b8fad157d0b878a2476cec998aba0643742bf/figs/2020-01-31_natural-earth-world-map.png)
- the nameprism groupings from Figure 1?

If the former, it's a bit confusing to have letters refer to different groupings. Could we just replace with the standard world geographical mapping region labels?

Author: My bad. The linked figure is incorrect. It should be fixed now.
